docs/releases/5.2.md

```diff
@@ -154,6 +154,31 @@ def register_frank_menu_item():
 
 ## Upgrade considerations - deprecation of old functionality
 
+### Legacy moderation system is deprecated
+
+The legacy moderation system, which was replaced by the new workflow system in Wagtail 2.10, is now deprecated. Since Wagtail 2.10, submitting a page for moderation will use the new workflow system. However, the legacy moderation system is still in place for approving and rejecting pages that were submitted for moderation before Wagtail 2.10.
+
+To view all pages that are still in the legacy moderation system backlog, you can sign in as a superuser and see if there is a "Pages awaiting moderation" section in the admin dashboard. You can approve or reject the pages from there. You can also do this programmatically by querying for `Revision.objects.filter(submitted_for_moderation=True)` and calling `revision.approve_moderation()` or `revision.reject_moderation()` on each revision.
```

Comment:
I added this paragraph @jacobtoppm, I believe it's enough?

I tried saving a new revision with `submitted_for_moderation=True`, which made the page show on the dashboard and I tried approving/rejecting from there. I also tried doing it programmatically and it seems to work.

Reply:
Yep, that's enough for me! I suspect the likely crossover is very small, but I think it's nicer to have a little pointer like this